README.md

```diff
@@ -1 +1 @@
-# hello
```

Comment:
Kindly note that you are required to use the [Microverse Readme template] as per the project requirements. (https://github.com/microverseinc/readme-template/blob/master/README.md) as per the requirements
<img width="1623" alt="Screenshot 2022-09-22 at 13 45 59" src="https://user-images.githubusercontent.com/48546992/191759480-eb38f712-08a8-4e76-be18-51eda3e34b54.png">

- Note that the read me explains to your users how the app works works.
- Also note that for subsequent projects, this is a cause for invalidation of your submission

Reply:
Hi @amexabee 
which is the last one you refer to?